examples/app_commands/info.py

```diff
@@ -1,45 +1,40 @@
+# Imports
 import discord
-from discord.ext import commands
-
-# imports
 
 intents = discord.Intents(
     guilds=True,
     members=True,
     messages=True,
 )
 
-bot = commands.Bot(
-    command_prefix="/",
-    description="An example to showcase how to extract info about users",
-    intents=intents,
-)
+bot = discord.Bot(intents=intents)
 
+@bot.command(name="userinfo", description="Gets information about an user.", guild_ids=[...])
+async def userinfo(ctx, *, user: discord.Member = None):
+    # If the user is not provided, it will default to the message author.
+    user = user or ctx.author
 
-@bot.slash_command(name="userinfo", description="gets the info of a user")
-async def info(ctx, user: discord.Member = None):
-    user = user or ctx.author  # if no user is provided it'll use the the author of the message
-    e = discord.Embed()
-    e.set_author(name=user.name)
-    e.add_field(name="ID", value=user.id, inline=False)  # user ID
-    e.add_field(
+    embed = discord.Embed(title=f"User Info for {user}")
+    embed.set_author(name=user.name)
+    embed.add_field(name="ID", value=user.id, inline=False) # ID of the user.
+    embed.add_field(
         name="Joined",
         value=discord.utils.format_dt(round(user.joined_at.timestamp()), "F"),
         inline=False,
-    )  # When the user joined the server
-    e.add_field(
+    )  # The timestamp of when the user joined the server.
+    embed.add_field(
         name="Created",
         value=discord.utils.format_dt(round(user.created_at.timestamp()), "F"),
         inline=False,
-    )  # When the user's account was created
-    colour = user.colour
+    )  # The timestamp of when the user created their account.
+    colour = user.color
     if colour.value:  # if user has a role with a color
-        e.colour = colour
+        embed.color = color
 
     if isinstance(user, discord.User):  # checks if the user in the server
-        e.set_footer(text="This member is not in this server.")
+        embed.set_footer(text="This user is not in this server.")
 
-    await ctx.respond(embed=e)  # sends the embed
+    await ctx.respond(embed=embed)  # Sends the embed
```

Comment:
```suggestion
    await ctx.respond(embeds=[embed])  # Sends the embed
```
`embed` is being deprecated in the Discord API in favor of using `embeds`

Reply:
`embed=` iirc converts it to a list and gives a `embeds` to discord, could be wrong tho

Reply:
It's still better to be explicit in usage so the user knows what's actually happening at a glance without having to guess if it's being converted or not.

Reply:
> It's still better to be explicit in usage so the user knows what's actually happening at a glance without having to guess if it's being converted or not.

it is kinda obvious tho that `embed=` would be 1 embed and `embeds=` would be a list

Reply:
The endpoint is `embeds` so we should be following that design.

Reply:
a lot of things in the library don't follow that principle

Reply:
@krittick also most if not all of the examples which need 1 embed only use `embed=`

Reply:
That doesn't mean we should continue following bad design just because it already exists. Discord themselves recommends using `embeds` and we should follow suit.

Reply:
Exactly.